### ISSUE TITLE
React 0.14

### ISSUE THREAD

**Issue by damienleroux** [mweststrate/mobservable-react#6]
Any plan to update mobservable-react to react 0.14?

I refer to:
```javascript
  // TODO: Fix in 0.14: React.findDOMNode is deprecated. Please use ReactDOM.findDOMNode from require('react-dom') instead.
                    var node = React.findDOMNode(this);
```

Thank you

**Comment by mweststrate** [comment]
Yup, so far I have been a bit reluctant to upgrade, since it is easier to
be forward compatible with 0.14 than backward compatible with 0.13, but
i'll upgrade later this week or next week anyway. Note that
mobservable-react should already work correctly with 0.14.

**Comment by damienleroux** [comment]
I experienced this error in chromium console:
``` 
[mobservable.view 'reactiveRender'] There was an uncaught error during the computation of ComputedObservable[reactiveRender:undefined] function reactiveRender() {
                        if (!hasRendered) {
                            hasRendered = true;
                            mobservable.extras.withStrict(true, function() {
                                rendering = baseRender.call(self);
                            });
                        } else {
                            self.__$mobDisposer();
                            React.Component.prototype.forceUpdate.call(self);
                        }
                    }ViewNode.computeNextState @ mobservable.js?8352:608
index.js?36c8:67 Uncaught ReferenceError: isPlainObject is not defined
```

After [futher reserach](https://github.com/justmoon/node-extend/issues/31), I finally find out that it was my version of extend that was not up to date. I used 2.0 instead of 3.0.

Thank you for your previous answer

**Comment by mweststrate** [comment]
Just published `mobservable-react@2.0.0` which uses React 0.14 instead of React 0.13.
